openfl-workspace/flower-app-pytorch/README.md

```diff
@@ -0,0 +1,291 @@
+
+```yaml
+task_runner:
+  defaults: plan/defaults/task_runner.yaml
+  template: openfl.federated.task.runner_flower.FlowerTaskRunner
+  settings:
+    auto_shutdown: True
+```
+3. `FlowerDataLoader` with similar high-level functionality to other dataloaders.
+
+**IMPORTANT NOTE**: `aggregator.settings.rounds_to_train` is set to 1. __Do not edit this__. The actual number of rounds for the experiment is controlled by Flower logic inside of `./app-pytorch/pyproject.toml`. The entirety of the Flower experiment will run in a single OpenFL round. The aggregator round is there to stop the OpenFL components at the completion of the experiment.
+
+4. `Task` - we introduce a `tasks_connector.yaml` that will allow the collaborator to connect to Flower framework via the local gRPC server. It also handles the task runner's `start_client_adapter` method, which actually starts the Flower component and local gRPC server. By setting `local_server_port` to 0, the port is dynamically allocated. This is mainly for local experiments to avoid overlapping the ports.
+
+```yaml
+tasks:
+  settings:
+    connect_to: Flower
+  start_client_adapter:
+    function: start_client_adapter
+    kwargs:
+      local_server_port: 0
+```
+
+## Running the Workspace
+Run the workspace as normal (certify the workspace, initialize the plan, register the collaborators, etc.):
+
+```SH
+# Generate a Certificate Signing Request (CSR) for the Aggregator
+fx aggregator generate-cert-request
+
+# The CA signs the aggregator's request, which is now available in the workspace
+fx aggregator certify --silent
+
+# Initialize FL Plan and Model Weights for the Federation
+fx plan initialize
+
+################################
+# Setup Collaborator 1 
+################################
+
+# Create a collaborator named "collaborator1" that will use shard "0"
+fx collaborator create -n collaborator1 -d data/1
+
+# Generate a CSR for collaborator1
+fx collaborator generate-cert-request -n collaborator1
+
+# The CA signs collaborator1's certificate
+fx collaborator certify -n collaborator1 --silent
+
+################################
+# Setup Collaborator 2 
+################################
+
+# Create a collaborator named "collaborator2" that will use shard "1"
+fx collaborator create -n collaborator2 -d data/2
+
+# Generate a CSR for collaborator2
+fx collaborator generate-cert-request -n collaborator2
+
+# The CA signs collaborator2's certificate
+fx collaborator certify -n collaborator2 --silent
+
+##############################
+# Start to Run the Federation
+##############################
+
+# Run the Aggregator
+fx aggregator start
```

Comment:
How about we point to our [quickstart](https://openfl.readthedocs.io/en/latest/tutorials/taskrunner.html) guide?
The quickstart covers context behind certain commands (when and where to run them), is CI-verified, and reduces duplication. If this requires contextual updates to the quickstart guide, we can discuss.

Reply:
I personally think it makes sense to have self-contained documentation for the Flower/OpenFL interop workspace, even if there's some duplication with the rest of the documentation.